CONTRIBUTING.md

```diff
@@ -194,3 +194,76 @@ IntelliJ may not pick up protoc generated classes as they can be very huge. If t
 
 ## CI
 The Ozone project uses Github Actions for its CI system.  The configuration is described in detail [here](.github/ci.md).
+
+## Debugging
+
+### Setup Ozone and Kubernetes
+
+Install a local server(k3s is recommended) and the Kubernetes command-line tool, kubectl. After setting up k3s and kubectl, build the project so that a SNAPSHOT image is created.
+
+1. Build the project `mvn clean install -DskipShade -DskipTests`
+2. `cd ozone/hadoop-ozone/dist/target/ozone-X.X..`
+3. Build the image `docker build -t apache/ozone:X.X-SNAPSHOT .`
+
+### Attach the debugger in IntelliJ
+
+Configure the debugger to attach to ports 5005, 6006, 7007.
```

Comment:
Make it clear that these are example ports, something like:

Configure the debugger to attach to ports selected for this purpose.  For example the ports 5005, 6006, 7007 are used to connect to the om, scm, and s3g.